packages/amplify_flutter/test/config/testdata/src/generate.js

```diff
@@ -0,0 +1,179 @@
+        awscloudformation: {
+            Region: REGION,
+        }
+    }
+};
+
+const authConfigJSON = fs.readFileSync('./aws_cognito_configuration.json');
+const authConfig = JSON.parse(authConfigJSON);
+
+const makeAppSyncConfig = (authType) => {
+    return {
+        service: 'AppSync',
+        output: {
+            authConfig: {
+                defaultAuthentication: {
+                    authenticationType: authType
+                }
+            },
+            GraphQLAPIEndpointOutput: GRAPHQL_ENDPOINT,
+            GraphQLAPIKeyOutput: authType === 'API_KEY' ? API_KEY : undefined,
+        }
+    };
+};
+
+const makeRestConfig = (authType) => {
+    return {
+        service: 'API Gateway',
+        output: {
+            RootUrl: REST_ENDPOINT,
+        },
+        authorizationType: authType,
+    }
+};
+
+const testVectors = [
+    {
+        name: 'Empty',
+        metadata: {
+            ...defaultMetadata
+        }
+    },
+    {
+        name: 'Analytics',
+        metadata: {
+            ...defaultMetadata,
+            analytics: {
+                test: {
+                    output: {
+                        Id: ANALYTICS_APP_ID,
+                        Region: REGION
+                    }
+                }
+            },
+        }
+    },
+    {
+        name: 'Auth',
+        metadata: {
+            ...defaultMetadata,
+            auth: {}
+        },
+        awsConfig: authConfig
+    },
+    {
+        name: 'API',
+        metadata: {
+            ...defaultMetadata,
+            api: {
+                API_KEY: {
+                    ...makeAppSyncConfig(API_KEY_AUTH_TYPE)
+                },
+                AWS_IAM: {
+                    ...makeAppSyncConfig(IAM_AUTH_TYPE)
+                },
+                REST: {
+                    ...makeRestConfig(COGNITO_AUTH_TYPE)
+                },
+            }
+        }
+    },
+    {
+        name: 'Storage',
+        metadata: {
+            ...defaultMetadata,
+            storage: {
+                S3: {
+                    service: 'S3',
+                    output: {
+                        BucketName: BUCKET,
+                        Region: REGION,
+                    }
+                }
+            }
+        }
+    }
+];
+
+let out = 
+    `//
+    // Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+    //
+    // Licensed under the Apache License, Version 2.0 (the "License").
+    // You may not use this file except in compliance with the License.
+    // A copy of the License is located at
+    //
+    //  http://aws.amazon.com/apache2.0
+    //
+    // or in the "license" file accompanying this file. This file is distributed
+    // on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+    // express or implied. See the License for the specific language governing
+    // permissions and limitations under the License.
+    //
+    
+    // GENERATED FILE. DO NOT MODIFY BY HAND.
```

Comment:
```suggestion
    // GENERATED FILE. DO NOT MODIFY BY HAND. SEE `./src/generate.js`.
```

Would be nice to point someone to the source of the generated code.

Reply:
Fixed